runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -1504,7 +1504,12 @@ Java_java_lang_invoke_MethodHandleNatives_getMemberVMInfo(JNIEnv *env, jclass cl
 					if (J9_ARE_ANY_BITS_SET(methodID->vTableIndex, J9_JNI_MID_INTERFACE)) {
 						vmindex = methodID->vTableIndex & ~J9_JNI_MID_INTERFACE;
 					} else if (0 == methodID->vTableIndex) {
-						vmindex = -1;
+						jint refKind = (flags >> MN_REFERENCE_KIND_SHIFT) & MN_REFERENCE_KIND_MASK;
+						if ((MH_REF_INVOKEVIRTUAL == refKind) || (MH_REF_INVOKEINTERFACE == refKind)) {
```

Comment:
Maybe add a comment for why virtual/interface have different vmindex value

Reply:
Comments added.